src/collection/collectionquery.cpp

```diff
@@ -118,6 +145,22 @@ CollectionQuery::CollectionQuery(const QSqlDatabase &db, const QString &songs_ta
 
 }
 
+QString CollectionQuery::RemoveSqlOperator(QString &token)
+{
```

Comment:
Move { to the previous line
Add one empty line after { and before } similar to the other code.